### ISSUE TITLE
Incoherent hierarchy error

### ISSUE THREAD

**Issue by bramantoleksono** [PyFPDF/fpdf2#631]
Hi, I am looking for a solution to render HTML to pdf using fpdf2 and it throws a Value Error with message `Incoherent hierarchy: cannot start a level 4 section after a level 0 one`. 
What is the issue, and how to solve the error. Please advise.

This is the code I use to produce the mentioned error.
```python
from fpdf import FPDF

pdf = FPDF()
pdf.add_page()
sample_html = """<body>
  <header>
    <section class="container">
      <h1 class="common_typo hero">H1</h1>
      <p class="common_typo">Text</p>
    </section>
  </header>
  <page>
    <div class="frame">
      <div class="field_container">
        <h5 class="common_typo header">H5</h5>
        <p class="common_typo">Text</p>
      </div>
  </page>
</body>"""
pdf.write_html(sample_html)
```

**Comment by Lucas-C** [comment]
Hi @bramantoleksono, and thank you for reporting this.

This is actually a bug in `fpdf2`: we should not be so strict regarding the order of headings, especially in `write_html()`.

I'll have a look at this asap

**Comment by Lucas-C** [comment]
This has been fixed, but not released yet.

You can test it using the latest version of `fpdf2` from the `master` branch of this repoitory:

	pip install git+https://github.com/PyFPDF/fpdf2.git@master

**Comment by Lucas-C** [comment]
FYI, this fix has been released 2.6.1

@allcontributors please add @bramantoleksono for bug
